lib/heroku-deflater/cache_control_manager.rb

```diff
@@ -20,9 +20,9 @@ def setup_max_age(max_age)
 
     def cache_control_headers
       if rails_version_5?
-        { 'Cache-Control' => cache_control }
+        { 'Cache-Control' => app.config.public_file_server.headers }
```

Comment:
@brian-kephart What about people that are using the 'Expires' header along with 'Cache-Control' though? They often go together. Something like this:
```ruby
  config.public_file_server.headers = {
    'Cache-Control' => "public, max-age=#{365.days.seconds.to_i}",
    'Expires' => "#{1.year.from_now.to_formatted_s(:rfc822)}"
  }
```